tpdatasrc/tpgamefiles/mes/damage.mes

```diff
@@ -0,0 +1,89 @@
+// damage string construction strings
+
+
+// damage description strings
+
+{100}{~Weapon~[TAG_WEAPONS]}
+{101}{Strength}
+{102}{Critical Hit}
+{103}{Unknown}
+{104}{Invulnerable}
+{105}{~Saving Throw~[TAG_SAVING_THROW_DESC]}
+{106}{~Sneak Attack~[TAG_CLASS_FEATURES_ROGUE_SNEAK_ATTACK]}
+{107}{~Saving Throw~[TAG_SAVING_THROW_DESC] [with ~Evasion~[TAG_CLASS_FEATURES_EVASION]]}
+{108}{~Saving Throw~[TAG_SAVING_THROW_DESC] [with ~Improved Evasion~[TAG_CLASS_FEATURES_IMPROVED_EVASION]]}
+{109}{~Saving Throw~[TAG_SAVING_THROW_DESC] [with ~Fire Shield~[TAG_SPELLS_FIRE_SHIELD]]}
+{110}{Spell [~Chill Shield~[TAG_SPELLS_FIRE_SHIELD]]}
+{111}{Spell [~Warm Shield~[TAG_SPELLS_FIRE_SHIELD]]}
+{112}{~Defensive Roll~[TAG_CLASS_FEATURES_ROGUE_DEFENSIVE_ROLL]}
+{113}{~Unarmed Strike~[TAG_WEAPONS_STRIKE_UNARMED]}
+{114}{Bite}
+{115}{Claw}
+{116}{Rake}
+{117}{Gore}
+{118}{Slap}
+{119}{Slam}
+{120}{Sting}    
+{121}{~Enhancement~[TAG_ENHANCEMENT_BONUS]}
+{122}{ (~Empowered~[TAG_EMPOWER_SPELL])}
+{123}{ (~Maximized~[TAG_MAXIMIZE_SPELL])}
+{124}{~Resistance to Energy~[TAG_SPECIAL_ABILITIES_RESISTANCE_TO_ENERGY]}
+{125}{Bane}
+{126}{~Damage Reduction~[TAG_SPECIAL_ABILITIES_DAMAGE_REDUCTION]}
+{127}{~Extraordinary Ability~[TAG_SPECIAL_ABILITIES]}
+{128}{Heat Damage}
+{129}{Doubled Damage}
+{130}{Blood Drain}
+{131}{Incorporeal Immunity}
+{132}{Immunity}
+{133}{Rend}
+{134}{Ethereal Immunity}
+{135}{Attacker is Ethereal}
+{136}{Vulnerable}
+
+{1000}{Unspecified}
+{1001}{Bludgeoning}
+{1002}{Piercing}
+{1003}{Slashing}
+{1004}{Bludgeoning and Piercing}
+{1005}{Piercing and Slashing}
+{1006}{Slashing and Bludgeoning}
+{1007}{Slashing and Bludgeoning and Piercing}
+{1008}{Acid}
+{1009}{Cold}
+{1010}{Electricity}
+{1011}{Fire}
+{1012}{Sonic}
+{1013}{Negative Energy}
+{1014}{Nonlethal}
+{1015}{Poison}
+{1016}{Positive Energy}
+{1017}{Force}
+{1018}{Blood Loss}
+{1019}{Magic}
+
+{2000}{--}
+{2001}{--}
+{2002}{~Silver~[TAG_SILVER]}
+{2003}{Magic}
+{2004}{Good}
+{2005}{Evil}
+{2006}{Chaos}
+{2007}{Law}
+{2008}{~Adamantine~[TAG_ADAMANTINE]}
+{2009}{Bludgeoning}
+{2010}{Piercing}
+{2011}{Slashing}
+{2012}{~Mithril~[TAG_MITHRAL]}
+{2013}{Cold Iron}
+{2014}{--}
+
+// Added by Sagenlicht
+{3000}{~Critical Strike~[TAG_SPELLS_CRITICAL_STRIKE]}
```

Comment:
Are you sure these even work? I think Temple+ looks up tpmes/damage.mes rather than mes/damage.mes

Reply:
I don't know :( I am used to have the files in the ToEE  overrides folder and then I have to place them in overrides/mes/damage.mes. 

I wasn't aware Temple+ requires a different file structure, I am sorry about that. I didn't find the time to work on this today, I will push an update tomorrow then.

Reply:
I added tpmes/damage.mes so as to not override the regular damage.mes. That was a long time ago however.
In general I think I'm going to have to bite the bullet and allow for composing mes files from different sources so I don't have to create extender handling for every single one.

Reply:
I only need the modifications in damage.mes because I can't use a string in damagePacket add_dice.

It would make more sense to me, to actually support strings for that in tpdp. I accidentally thought you have done that. I personally would not need a modified damage.mes if custom strings would be supported.

A whole different story would be the radial.mes. I have to modify it for radial options in spells. We talked about that in the doc in length, you said you add it to the to do list.

Reply:
Yeah that's what I mean. There's a ton of mes files and the list of extenders keeps growing. I want to solve this at the root.

Reply:
Ok, just to be sure, we are on the same page.
I move 
> damage.mes 
spell_radial_menu_options.mes 
generell_help_mods.tab (contains the modifactions to help)

to co8infra/tpmes (in this folder is help_extensions.tab atm)

and keep the rest of the files in tpgamefiles? Which would be:
>buff and debuff symbols
spell_compendium_help.tab (contains the help to all new spells)
spell_compendium_spell.mes (short description) (though I am unsure if I need a subfolder in mes)
spell_compendium_spell_long_descriptions.mes (long descrption)
blackguard and assassin char_class files
spell_compendium_partsys.tab
spell_compendium_spell_enum.mes
spell_compendium_sounds.mes

Reply:
Not sure about tpmes. I'm on vacation now, will get back to you next week.